src/index.js

```diff
@@ -111,7 +111,8 @@ module.exports.anonymizer = (
 
       if (blacklistPaths.test(path) || !whitelistPaths.test(path)) {
         if (trim && replacedValue === DEFAULT_REPLACEMENT) {
-          const path = this.path.map(value => (isNaN(value) ? value : '[]'));
+          // eslint-disable-next-line no-confusing-arrow
+          const path = this.path.map(value => isNaN(value) ? value : '[]');
```

Comment:
Why do we need the change AND the comment? Are we sure that this maintains the same logic?

Reply:
Well, `no-confusing-arrow` is used to prevent my change :). Don't know why, eslint was marking it as an error. Already changed it.